storage/src/state/tests.rs

```diff
@@ -346,3 +347,29 @@ fn test_transaction_fees() {
     assert_eq!(new_coinbase_record.value(), expected_block_reward);
     assert_eq!(output_record.value(), amount);
 }
+
+#[test]
+fn test_get_blocks_iterator() {
+    // Initialize a new temporary directory.
+    let directory = temp_dir();
+
+    // Initialize an empty ledger.
+    let ledger_state = LedgerState::open_writer::<RocksDB, _>(directory.clone()).expect("Failed to initialize ledger");
+
+    // Read the test blocks;
+    // note: they don't include the genesis block, as it's always available when creating a ledger.
+    let test_blocks = fs::read("benches/blocks_1").unwrap_or_else(|_| panic!("Missing the test blocks file"));
```

Comment:
You can use expect in this case.
```suggestion
    let test_blocks = fs::read("benches/blocks_1").expect("Missing the test blocks file");
```